wire-ios-data-model/Tests/MLS/ProteusToMLSMigrationCoordinatorTests.swift

```diff
@@ -60,6 +66,52 @@ class ProteusToMLSMigrationCoordinatorTests: ZMBaseManagedObjectTest {
         super.tearDown()
     }
 
+    // MARK: - Tests
+
+    func testMigrateOrJoinGroupConversations_CallsJoinGroupForNewConversations() async throws {
+        // GIVEN
+        let selfUser = ZMUser.selfUser(in: syncMOC)
+        selfUser.teamIdentifier = .init()
+        let conversation = ZMConversation.insertNewObject(in: uiMOC)
+        conversation.mlsGroupID = .random()
+        let groupID = try XCTUnwrap(conversation.mlsGroupID)
+        let mockMLSService = MockMLSService()
```

Comment:
no need to create a new `MockMLSService` here. It's already created in `setUp`

Reply:
Fixed it!